l10n_it_pec_messages/model/mail_message.py

```diff
@@ -115,6 +142,67 @@ def _get_out_server(self, cr, uid, ids, name,
         'direction': 'in'
     }
 
+    def CheckStatus(self, cr, uid, ids, context=None):
+        mail_mail_pool = self.pool['mail.mail']
+        if context is None:
+            context = {}
+        if not hasattr(ids, '__iter__'):
+            ids = [ids]
+        error_lst = []
+        receipt_list = []
+        completed = True
+        res = {}
+        if ids:
+            for message in self.browse(cr, uid, ids, context=context):
+                mail_sent_ids = mail_mail_pool.search(
+                    cr,
+                    uid,
+                    [('mail_message_id', '=', message.id)],
+                    context=context
+                )
+                if not mail_sent_ids:
+                    _logger.exception(
+                        'No sent mail for message %s \
+                        from %s server %s.', message.id)
```

Comment:
Non è chiaro il significato di ```CheckStatus```: se è un controllo generale su un record di mail.message, non è vero che ogni mail.message deve sempre avere un mail.mail collegato. Immagino che questo controllo serva in casi particolari, quindi bisogna esplicitare nel suo nome il suo significato.
Di conseguenza, anche la sua chiamata da res.partner deve essere chiaro a cosa serva